ansible/eos_install_config

```diff
@@ -42,26 +45,39 @@ requirements:
 options:
     hostname:
         description: IP or FQDN of the device you want to connect to
-        required: true
+        required: True
     username:
         description: Username
-        required: true
+        required: True
     password:
         description: Password
-        required: true
+        required: True
     use_ssl:
```

Comment:
Ok, disregard. I was thinking in napalm. I personally don´t think we should support/provide this plugin at all. I put it there as a proof of concept but I am not planning to use it so I don´t want to spend time or effort in working on that. But if you want to... feel free ; )